xml/GenCode.php

```diff
@@ -14,7 +14,8 @@
 }
 
 if ($memLimit >= 0 and $memLimit < 536870912) {
-  ini_set('memory_limit', '512M');
```

Comment:
The reason for the check is that if you have 2GB of ram it will reduce to 512MB without the check...